pkg/clusteroperator/clusteroperator_test.go

```diff
@@ -148,12 +149,123 @@ func TestClusterOperatorGet(t *testing.T) {
 	}
 }
 
+func TestHasDesiredVersion(t *testing.T) {
```

Comment:
Please re-name to:
**TestClusterOperatorHasDesiredVersion**

Reply:
Re-named, thanks